test/channels/application_cable/connection_test.rb

```diff
@@ -1,13 +1,15 @@
 # frozen_string_literal: true
 
-require "test_helper"
```

Comment:
If you are using `rspec` you do not need `test/` so you can remove them. All your tests will be located in `spec/` directory